test/markup/shell/command-continuation-alt.txt

```diff
@@ -0,0 +1,5 @@
+> /bin/cat \.travis.yml\
+ -b | head -n1
+     1	language: node_js
+> echo 'hello'
+hello
```

Comment:
How is this diff from the first one?  If there are just two tests for the same thing you don't need a second file, just make one larger test.

Reply:
There's no space before `\` (line continuation) and there's a `\` as part of the command (not at the end of the line).
I also added a simple command at the end to make sure that the regular expression is not too greedy.

I merged everything in `command-continuation.txt`.